hasktorch/hasktorch.cabal

```diff
@@ -76,6 +76,8 @@ library
                     , safe-exceptions
                     , zlib >= 0.6
                     , inline-c
+ default-extensions:  Strict
+                    , StrictData
```

Comment:
@apaszke 
I add Strict pragma to not use laziness.
https://gitlab.haskell.org/ghc/ghc/-/wikis/strict-pragma